stable/kube-state-metrics/Chart.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
 - monitoring
 - prometheus
 - kubernetes
-version: 2.7.2
+version: 2.7.3
```

Comment:
If we are  changing the default flags, then this would be minor version update IMO

Reply:
Thanks, changed to 2.8.0